src/CorrelationsReportView.ts

```diff
@@ -38,14 +39,15 @@ export default class CorrelationView extends ItemView {
 		this.plugin = plugin;
 		this.lower = 0.5;
 		this.medium = 0.75;
-		this.max = Infinity;
```

Comment:
why this change?

Reply:
Ahh yes. Because a number input element doesn't consider Infinity to be a valid value

Reply:
Ah, hm. I wonder how we can get it to show all correlations then. Because this way, it'll skip anything with an Inf or -Inf correlation, right? Do we want the?

Reply:
Ah. But then what about correlations that _do_ contain "Infinity" and "-Infinity" values? Are we okay just not showing them?